static/src/styles/landing/LandingUpdateFeed.styl

```diff
@@ -1,14 +1,34 @@
 .LandingUpdateFeed {
+    padding: grid(3);
+}
+
+.LandingUpdateFeed__postContainer {
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
-  justify-content: flex-start;
+  justify-content: space-between;
   align-items: stretch;
-  padding: grid(2);
 }
 
 .LandingUpdateFeed__item {
-  padding: grid(2);
-  flex: 1 1 auto;
-  width: grid(40);
+  margin-bottom: grid(2);
+  flex-basis: 24%;
```

Comment:
Design nit: Percentage based gutters lead to the space between elements being different to the space around them. Furthermore, as there is also a fixed margin here (2 units), the horizontal spacing differs from the vertical spacing.